engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -4128,6 +4129,7 @@ public void mainHelper() throws Exception
 
   public static void main(final String[] args) throws Exception
   {
+    LoggerUtil.addAppender("apexclient");
```

Comment:
For Cli, I don't think it should be using the property name LogicalPlan.LOGGER_APPENDER.getName(). That could conflict with the LOGGER_APPENDER that you are trying to specify for the running application (aka stram and streaming containers)